src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1048,7 +1048,14 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
                                     ->end()
                                     ->scalarNode('tags')->defaultNull()->end()
                                     ->booleanNode('public')->defaultFalse()->end()
-                                    ->integerNode('default_lifetime')->end()
+                                    ->integerNode('default_lifetime')
+                                        ->info('Default lifetime of the pool')
+                                        ->example('"600" for 5 minutes expressed in seconds or "PT5M" for five minutes expressed as ISO 8601 time interval')
+                                        ->beforeNormalization()
+                                            ->ifTrue(\Closure::fromCallable('is_string'))
+                                            ->then(function ($lifetime) { return (int) \DateTime::createFromFormat('U', 0)->add(new \DateInterval($lifetime))->format('U'); })
```

Comment:
be careful about cases where the string is an env placeholder. This PR currently drops support for using an env placeholder in this config setting.

Reply:
Dont all dateinterval formats starts with `"P"`?

If so, cant we just check for that instead of doing `ifString()`?

Reply:
Yup, that’s what I do now